docs/tutorials/dashboard.mdx

```diff
@@ -0,0 +1,147 @@
+
+To connect the peer to the room, you can either press the **Connect** button in the peer's row, or scan the QR code with your mobile app, and connect there.
+
+### Sending tracks
+
+When the peer connects to the room, you will see the track menu. It allows you to send tracks to the room.
+![Track menu](/img/tutorials/dashboard/dashboard_track_menu.png)
+
+You can select the _audio or video tracks_ You want to send to the room. Apart from your camera/microphone,
+there is an option to pick one from a list of _mock tracks_ that we prepared.
+
+You can also select the _resolution_ of the video track
+or if you want, send the video as a _simulcast_ stream.
+
+There is also an option to add custom _metadata_ (in JSON format) to the track. The peer will send metadata with the track to the room. Each recipient will be able to see it.
+
+When you select the track, click the **Add track** button. If everything is correct, client will send the track to the room and other peers
+will be able to see and hear you.
+
+### Receiving tracks
+
+When the peer connects to the room, it will automatically receive tracks from other peers.
+
+#### Congrats! The basic utils of the dashboard are now covered. Next, we will dive into useful features that will help you with testing.
+
+## A ton of information that the dashboard provides
+
+### Server logs in console
+
+On the side menu, you can set up the dashboard to show the server logs in the console.
+Logs are great for debugging purposes and present the variety of communication that takes place between the dashboard and the backend.
+You can also register your callbacks in your Jellyfish projects.
+
+
+<ImageWrapper
+    url={"/img/tutorials/dashboard/dashboard_server_logs.png"}
+    height={500}
+    alt="Server logs"
+/>
+
+
+### Server/Room/Client state
+
+Each server, room, and client has its state. You can see the state of each of them in the dashboard by clicking appropriate buttons.
+![Server state](/img/tutorials/dashboard/dashboard_states.png)
+
+### Server events
+
+Apart from the state, you can also see the events that occur on the server.
+You can see them after clicking the **Show Server Events** button.
+
+![Server events](/img/tutorials/dashboard/dashboard_server_events.png)
+
+## HLS and RTSP streaming
+
+The dashboard allows adding HLS and RTSP streams in the room. You can set up the streams and see them in the room. It will look like this:
+![HLS and RTSP](/img/tutorials/dashboard/dashboard_hls_rtsp.png)
+
+## Loadbalancing in Dashboard
+
+Jellyfish Server supports load balancing.This means you can run multiple instances of the server and use them simultaneously to minimize the load on each of them.
+The dashboard reflects this feature. If you run multiple Jellyfish instances (Docker compose does that by default), connect one server to the dashboard.
+When a server opens a room on a different instance, the dashboard will automatically connect to it and display the room.
```

Comment:
Really nice :+1: 

Two suggestions:
* could we link docker-compose we are talking about
* maybe let's explain how it is possible that dashboard automatically sees another Jellyfish instance? I mean that we could mention that Jellyfishes run in a cluster and when JFD asks for a new room it's created on JF with the lowest load and it's IP of JF where the new room was created is returned to the JFD or something like this?